### PR TITLE
Add graph components to main chart

### PR DESCRIPTION
**Pull request by rokroskar**
Webhook and triples service works with the helm command found in `charts/README.md`. 

~~ingress for Jena doesn't work because Jena is not running on a sub-path. Unclear at the moment how to fix this, presumably there is some way to set http server options.~~

We aren't exposing Jena via ingress for the time being. Services talk to Jena via cluster routing. 

contingent on https://github.com/SwissDataScienceCenter/renku-graph/pull/26
uses https://github.com/SwissDataScienceCenter/renku-graph/pull/32

### CODE REVIEW

File: charts/renku/requirements.lock [hunk 1]
```diff
@@ -11,6 +11,9 @@ dependencies:
 - name: gitlab
   repository: https://swissdatasciencecenter.github.io/helm-charts/
   version: 0.3.0
+- name: renku-graph
+  repository: file://../../../renku-graph/helm-chart/renku-graph
```
Comment (leafty) [inline charts/renku/requirements.lock]: ?

File: charts/renku/requirements.yaml [hunk 2]
```diff
@@ -15,6 +15,10 @@ dependencies:
   repository: "https://swissdatasciencecenter.github.io/helm-charts/"
   version: "0.3.0"
   condition: gitlab.enabled
+- name: renku-graph
+  repository: file://../../../renku-graph/helm-chart/renku-graph
```
Comment (leafty) [inline charts/renku/requirements.yaml]: ?

File: scripts/minikube_deploy.py [hunk 3]
```diff
@@ -64,49 +72,89 @@ def main():
...
-            '--set', 'notebooks.jupyterhub.hub.extraEnv.GITLAB_URL=http://{mip}/gitlab'.format(mip=minikube_ip()),
-            '--set', 'notebooks.jupyterhub.hub.services.gateway.oauth_redirect_uri=http://{mip}/api/auth/jupyterhub/token'.format(mip=minikube_ip()),
-            '--set', 'notebooks.gitlab.registry.host=10.100.123.45:8105',
-            '--set', 'gitlab.registry.externalUrl=http://10.100.123.45:8105/',
-            '--timeout', '1800',
+            '--namespace',
+            namespace,
+            '-f',
+            os.path.join(renku_chartpress_dir, 'minikube-values.yaml'),
+            '--set',
```
Comment (leafty) [inline scripts/minikube_deploy.py]: if we're expanding this, let's use `set-string`

**Comment by jachro** [comment]
With my minimal knowledge of helm I can say that all looks good.

**Comment by leafty** [comment]
For jena, it might work to just route to a single dataset endpoint like `jena-renku`

**Comment by rokroskar** [comment]
@leafty:
> For jena, it might work to just route to a single dataset endpoint like jena-renku

yes, that's what I was looking at as well. But I think there is a way to specify the base URL in the global config. We may end up with more datasets in the future which would make this a pain.

File: charts/renku/requirements.lock (hunk 1 above)
Comment (rokroskar) [reply charts/renku/requirements.lock]: it is, I forgot to put back the registry URL

**Comment by leafty** [comment]
From what I got from the fuseki docs is that there are a bunch of RDF files for configuration :roll_eyes:

**Comment by rokroskar** [comment]
yes, it's rather opaque for me atm. `prefix` means rdf prefix but also (I think) URL prefix...

File: scripts/minikube_deploy.py (hunk 3 above)
Comment (rokroskar) [reply scripts/minikube_deploy.py]: I didn't realize that was an option -- fixed in c7d0cf2

File: charts/minikube-values.yaml [hunk 4]
```diff
@@ -41,6 +41,7 @@ gitlab:
   demoUserIsAdmin: true
 
 notebooks:
+
```
Comment (jirikuncar) [inline charts/minikube-values.yaml]:
```suggestion
```

**Pull request merged**